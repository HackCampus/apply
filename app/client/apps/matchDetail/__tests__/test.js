const test = require('ava')
const sinon = require('sinon')

// const app = require('../app')
//
// test('init', t => {
//   t.snapshot(app.init())
// })
//
// test('view - initial', t => {
//   const {model} = app.init()
//   const dispatch = sinon.stub()
//   const view = app.view(model, dispatch)
//   t.snapshot(view.toString())
//   t.false(dispatch.called)
// })
