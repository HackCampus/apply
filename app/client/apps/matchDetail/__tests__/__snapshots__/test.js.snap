exports[`init 1`] = `"{"model":{"children":{}},"effect":null}"`;

exports[`view - initial 1`] = `""<div class="matchDetail">
        hello from matchDetail :)
      </div>""`;
